env/custom-ns-columns-cna.sh

```diff
@@ -0,0 +1,2 @@
+export CBIOPORTAL_URL="${CBIOPORTAL_URL:-http://localhost:8080}"
```

Comment:
remote these two exports and just have:
```export BACKEND=BasLee:custom-ns-columns-cna```

Reply:
changed in [1c4e19c](https://github.com/cBioPortal/cbioportal-frontend/commit/1c4e19c334deca6d2af2f49810eb9755e5ac0944)